pg2kinesis/__main__.py

```diff
@@ -32,9 +51,12 @@
               help='Attempt to on start create a the slot.')
 @click.option('--recreate-slot', default=False, is_flag=True,
               help='Deletes the slot on start if it exists and then creates.')
+@click.option('--operations', default='a', type=parse_operations_param,
```

Comment:
I think we can use built in `choice` type in `click` with `multiple` set to `true`, instead of needing this custom type.

See: https://stackoverflow.com/questions/37486367/click-choice-for-multiple-arguments

Reply:
I used the custom parser so we could do something more streamlined like `--operations ui` instead of `--operations update insert`, but i can change